CHANGELOG.rst

```diff
@@ -26,6 +26,10 @@
   including removal of the obsolete ``_pytest.assertion.oldinterpret`` module.
   Thanks `@nicoddemus`_ for the PR (`#1226`_).
 
+* Comparisons now always show up in full when 'CI' is found in the environment,
```

Comment:
@The-Compiler you think we should also mention that `BUILD_NUMBER` is considered, or be a little vague instead about "in Continuous Integration environments"?